docs/next/guides/accessibility/keyboard-shortcuts.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Keyboard shortcuts
+metaTitle: Keyboard shortcuts - Guide - Handsontable Documentation
+permalink: /next/keyboard-shortcuts
+canonicalUrl: /keyboard-shortcuts
+tags:
+  - key bindings
+  - keymap
+  - shortcut manager
+  - keyboard navigation
+---
+
+# Keyboard shortcuts
+
+Use and manage Handsontable's keyboard shortcuts.
+
+[[toc]]
+
+## About keyboard shortcuts
+
+You can intuitively navigate Handsontable with a keyboard, using the out-of-the-box [default keyboard shortcuts](#default-keyboard-shortcuts).
+
+You can also customize the entire set of keyboard shortcuts, by:
+- [Removing default keyboard shortcuts](#removing-default-keyboard-shortcuts)
+- [Replacing default keyboard shortcuts](#replacing-default-keyboard-shortcuts)
+- [Adding custom keyboard shortcuts](#adding-custom-keyboard-shortcuts)
+
+## Default keyboard shortcuts
+
+By default, Handsontable features the following keyboard shortcuts:
+
+::: details Default keyboard shortcuts
```

Comment:
The list of built-in shortcuts is very important and shouldn't be hidden in an expandable detail.